lib/ExpensiMark.js

```diff
@@ -576,4 +570,4 @@ export default class ExpensiMark {
         }
         return replacedText;
     }
-}
+}
```

Comment:
What's this change?
Same for another file

Reply:
oh my bad, I fixed